pkg/injector/patcher/sidecar_container.go

```diff
@@ -84,10 +87,6 @@ func (c *SidecarConfig) getSidecarContainer(opts getSidecarContainerOpts) (*core
 		args = append(args, "--control-plane-address", c.OperatorAddress)
 	}
 
-	if c.SentryAddress != "" {
```

Comment:
Why was this removed? Can't SentryAddress be empty?

Reply:
No- why would the sentry address ever be empty?

Reply:
It can be empty though, right? In standalone mode daprd runs without sentry. So it's _possible_ even if unlikely.

Reply:
Right, but this injector patching would not be happening in standalone mode.

Reply:
Correct. But it's possible to use daprd without sentry. So when someone uses the injector as a library, they have the choice.